### ISSUE TITLE
Run #24 notebook in Voila

### ISSUE THREAD

**Issue by SaOgaz** [spacetelescope/dat_pyinthesky#51]
#24

Jira Issue:
https://jira.stsci.edu/browse/DATJPP-8

**Comment by pllim** [comment]
I don't think this is possible as I can't find any notebook by @javerbukh that addresses #24 . There is nothing to run `voila` on for this.
